walkthroughs/1-integrating-event-and-api-driven-apps/walkthrough.adoc

```diff
@@ -67,10 +68,12 @@ image::images/arch.png[integration, role="integr8ly-img-responsive"]
 
 // tag::creating-api-connector[]
 
-
 [id='creating-api-connector_{context}']
 [.integr8ly-docs-header]
-=== Creating an API Connector 
+=== Creating an API Connector
+
+Fuse Online provides connectors for many services, for example, Gmail and IRC.
+However, each API must be registered as a separate customization.
```

Comment:
I found this a little unclear. "each API" means my own REST APIs? "Customisation" sounds intimidating 😄 

Maybe "Connecting to your own REST APIs requires registering them as a _Customisation_. This simply requires you to provide an OpenAPI definition for your API."

Reply:
"your own" might also be better as just "other" since it might not necessarily be owned by the user

Reply:
To create a connector for the RESTful API, you register the OpenAPI definition as a Customization.